package.json

```
@@ -6,6 +6,9 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.2.1",
+    "bootstrap": "^4.6.0",
```

Bootstrap `4.6.0` でハンバーガーメニューが動作しない不具合が発生していますので，

```suggestion
    "bootstrap": "^4.5.1",
```

に変更し，

```
yarn install --check-files
```

を実行して再度プッシュしなおして下さい。